bread/layout/base.py

```diff
@@ -67,6 +67,26 @@ def resolve(self, context, element):
         return format_value(value)
 
 
+class ObjectFieldLabel(hg.Lazy):
+    def __init__(self, fieldname):
+        self.fieldname = fieldname
+
+    def resolve(self, context, element):
+        return context["object"]._meta.get_field(self.fieldname).verbose_name
```

Comment:
For this one I recommend to take a look at ```fieldlabel``` in the same file. The difference is, that we sometimes want to display a field of a related object, e.g. ```object.primary_email_address.type```. Or we want to display the value of a property on the model but still get a good default label.

Reply:
I am not sure I understand what you meant, but I am now calling the `fieldlabel` method.

Reply:
Ok, good.

Yeah, sorry, not very good description. Let me explain: If you have something like

```@property
def age(self):
    return today - self.date
```

 as a model method then your initial version would not work when trying to display the label for ```age```, it works exlusively on database fields. But the point is, there are many cases where you want to have a dynamically calculated value (be it via ```queryset.annotate``` or just a method of the Model class) or an attribute of a field's value, not the value directly (that was the primary email address type example).
That is where the ```resolve_modellookup``` inside ````fieldlabel``` tries to get the correct type, callable or value. And then on *that* the ```verbose_name```